Calibration/HcalCalibAlgos/macros/CalibSort.C

```diff
@@ -418,6 +432,20 @@ void sort(std::vector<record>& records, bool debug) {
   }
 }
 
+void sortNew(std::vector<record>& records, bool debug) {
```

Comment:
I kept them to see the consistency in the 2 methods. In the long run I shall get rid of the old one.